pages/index.js

```diff
@@ -66,59 +66,67 @@ const styles = theme => ({
+    }
+  }
 
-  return (
-    <div className={classes.root}>
-      <Head>
-        <title>Material-UI</title>
-      </Head>
-      <div className={classes.hero}>
-        <div className={classes.content}>
-          <img
-            src="/static/images/material-ui-logo.svg"
-            alt="Material-UI Logo"
-            className={classes.logo}
-          />
-          <div className={classes.text}>
-            <Typography variant="display2" component="h1" color="inherit" gutterBottom>
-              {'Material-UI'}
-            </Typography>
-            <Typography
-              variant="headline"
-              component="h2"
-              color="inherit"
-              className={classes.headline}
-            >
-              {"React components that implement Google's Material Design."}
-            </Typography>
-            <Button
-              component={buttonProps => (
-                <Link
-                  variant="button"
-                  prefetch
-                  href="/getting-started/installation"
-                  {...buttonProps}
-                />
-              )}
-              className={classes.button}
-              variant="raised"
-            >
-              {'Get Started'}
-            </Button>
+  render() {
+    const classes = this.props.classes;
+
+    return (
+      <div className={classes.root}>
+        <Head>
+          <title>Material-UI</title>
+        </Head>
+        <div className={classes.hero}>
+          <div className={classes.content}>
+            <img
+              src="/static/images/material-ui-logo.svg"
+              alt="Material-UI Logo"
+              className={classes.logo}
+            />
+            <div className={classes.text}>
+              <Typography variant="display2" component="h1" color="inherit" gutterBottom>
+                {'Material-UI'}
+              </Typography>
+              <Typography
+                variant="headline"
+                component="h2"
+                color="inherit"
+                className={classes.headline}
+              >
+                {"React components that implement Google's Material Design."}
+              </Typography>
+              <Button
+                component={buttonProps => (
+                  <Link
+                    variant="button"
+                    prefetch
+                    href="/getting-started/installation"
+                    {...buttonProps}
+                  />
+                )}
+                className={classes.button}
+                variant="raised"
+              >
+                {'Get Started'}
+              </Button>
+            </div>
           </div>
         </div>
+        <HomeSteps />
+        <HomeBackers />
+        <HomeFooter />
       </div>
-      <HomeSteps />
-      <HomeBackers />
-      <HomeFooter />
-    </div>
-  );
+    );
+  }
 }
 
-PageHome.propTypes = {
+HomePage.propTypes = {
```

Comment:
It wasn't a typo 😆. It was a prefix to find all the page components. It's a convention we use at the office. But why not.